toplevel/colors.ml

```diff
@@ -0,0 +1,79 @@
+(************************************************************************)
```

Comment:
This file seems to be about calling topfmt apis, maybe it should be inlined in topfmt?

Reply:
I was wrong, coqargs is in sysinit/ and it is really about arguments common to most components (it looks weird to put in there something called `parse_extra_colors`.

I'd leave the file where it is in this PR, if you want to chop it, I'm not against, but does not seem to bring much to me.